.lintstagedrc

```diff
@@ -0,0 +1,7 @@
+{
+  linters: {
+    "*.{js,jsx,json,yml,yaml,css,less,scss,md,graphql,mdx}": "npm run format",
```

Comment:
💡 : lintstaged files can take an array of commands, and so by running npm run format, a lot of files will be modified, and then the developer would have to git add them manually again, so my suggestion would be to modify this `npm run format` to `["npm run format", "git add"]`

Reply:
Great suggestion, added! https://github.com/telus/create-library/pull/52/commits/91763ac6a4b1fe6522ba76eb5aeca1c31589914c Thank you!